lib/surveyor/models/response_set_methods.rb

```diff
@@ -64,11 +64,6 @@ def to_csv(access_code = false, print_header = true)
         end
       end
 
-      def as_json(options = nil)
-        template_paths = ActionController::Base.view_paths.collect(&:to_path)
-        Rabl.render(self, 'surveyor/show.json', :view_path => template_paths, :format => "hash")
-      end
```

Comment:
Sorry -- I may have missed this -- we totally use this in our app so I guess we can't actually remove `rabl`....

Reply:
WAIT we may override this. Let me check/

Reply:
I CHECKED we do in the answer and question models at least

Reply:
So we do override this for `survey` but not `response_set`. My intuition is that we're not using `to_json` on response sets but we should verify that before removing this. Were you able to run through the researcher and participant workflows after removing this?

Reply:
yeah i think we override it for all of the survey-related models

Reply:
As a P I went through and filled out a survey and submitted it just fine- and I was able to see it on the P's profile as the researcher. I was also able to edit and save a survey as a researcher

Reply:
actually hold the phone my local was pointing to a remote commit that is out of date 🤦

Reply:
OMG OMG OMG I'm so excited to see what you find!

Reply:
OK I went through this again and it seemed fine when pointed to my local version- I even double checked on the admin side looked at P survey responses